architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,17 +175,58 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
+//from https://blog.openshift.com/image-streams-faq/
+An _image stream is a pointer to an image using various identifying qualities. This means that even if the source image changes, 
+the image stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+Note that an image stream can be configured to periodically chack for updates (using the `--scheduled` tag) or you can start a 
+new build manually by re-importing an image (using the `oc import-image` command). 
```

Comment:
or you can manually re-import an image

It has nothing to do with builds, here.